TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -1304,6 +1304,53 @@ can increase your binary size and adversely affect performance.
   because of the larger symbol table slowing dyld down.
 -->
 
+### preconcurrency
+
+Apply this attribute to a declaration,
+to suppress strict concurrency checking.
+
+On an import declaration,
+this attribute reduces the strictness of concurrency checking
+for code that uses types from the imported module.
+Specifically,
+types from the imported module
+that aren't explicitly marked as either sendable or nonsendable
+can be used in a context that requires sendable types.
+
+On other declarations,
+this attribute reduces the strictness of concurrency checking
+for code that uses the symbol being declared.
+When you use this symbol in a scope that has  minimal concurrency checking,
+you can use members of this type
+that aren't explicitly marked as either sendable or nonsendable
+in a context that requires sendable types.
```

Comment:
Tech reviewer: Please confirm.  I'm putting together facts from a few places in the SE proposal, and I'm not sure this is correct.  Elsewhere, it says that `@preconcurrency` on a declaration for a nominal type generates code that interoperates with Swift 5 binaries.

Reply:
I don't think that's quite right. I think I would describe it more like this:

> On other declarations, `@preconcurrency` means that the declaration has been modified from its original version to add `Sendable` constraints or other concurrency-related attributes. Swift will generate code that is compatible with versions of the library that predated those additions, and it will not enforce the declaration's concurrency properties in scopes that have minimal concurrency checking.